faqs/galaxy/datasets_job_status.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Understanding job statuses
+description: Job statuses will help you understand the stage of your working.
+area: datasets
+layout: faq
+box_type: tip
+contributors: [garima]
```

Comment:
```suggestion
contributors: [garima, jennaj]
```

Reply:
Thanks for mentioning, will surely do!

Reply:
This is the cause of the failing test. Please add that change in this PR and then tests will pass :)

Reply:
Oh, okay, thanks again. Will do it shortly!